contracts/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 #![allow(clippy::pub_underscore_fields, clippy::module_name_repetitions)]
 #![cfg_attr(not(feature = "std"), no_std, no_main)]
 extern crate alloc;
+extern crate core;
```

Comment:
Are you sure we need `core`? It's my understanding that this is always imported.

Reply:
weird how it appeared..